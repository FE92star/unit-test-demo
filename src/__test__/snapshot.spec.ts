it('will check the matchers and pass', () => {
  const user = {
    createdAt: new Date(),
    id: Math.floor(Math.random() * 20),
    name: 'LeBron James',
  }

  expect(user).toMatchSnapshot({
    createdAt: expect.any(Date),
    id: expect.any(Number),
  })
})
