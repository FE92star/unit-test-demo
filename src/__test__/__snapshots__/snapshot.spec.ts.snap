// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`will check the matchers and pass 1`] = `
Object {
  "createdAt": Any<Date>,
  "id": Any<Number>,
  "name": "LeBron James",
}
`;
